src/components/Filter/ActiveFilterItem.js

```diff
@@ -1,12 +1,14 @@
 import React, {PropTypes} from 'react';
-import {capitalize} from '../../util';
+import {capitalize, getLanguageName} from '../../util';
 import styles from './ActiveFilterItem.scss';
 import withStyles from 'isomorphic-style-loader/lib/withStyles';
 import Button from 'react-bootstrap/lib/Button';
 import Glyphicon from 'react-bootstrap/lib/Glyphicon';
 
 const ActiveFilterItem = (props) => {
-  const tagItem = capitalize(props.tagItem);
+  let tagItem = props.groupName === 'language' ?
+   getLanguageName(props.tagItem) : props.tagItem;
```

Comment:
Dersom vi skal sjekke for `groupName==='language'` spesifikt, så vil jeg at det bare skal gjøres ett sted i hele CC-viewer, ikke både for ActiveFilterItem og for FilterGroups (hvis mulig).